public/pages/CreateTrigger/components/Action/actions/Message.js

```diff
@@ -122,6 +123,47 @@ const Message = ({
       </EuiFormRow>
 
       <EuiSpacer size="s" />
+
+      <EuiFormRow
+          label={
+            <div>
+              <span>Action throttling</span>
+              <EuiButtonEmpty
+                size="s"
+                onClick={() => { setFlyout({ type: 'messageFrequency' }); }}
+              >
+                Info
+              </EuiButtonEmpty>
+            </div>
+          }
+          style={{ maxWidth: '100%' }}
+        >
+          <EuiFlexGroup alignItems="center">
+            <EuiFlexItem grow={false} style={{ marginRight: '0px' }}>
+              <FormikSwitch
+                name={`actions.${index}.throttle_enabled`}
+                inputProps={{ label: 'Maximum Frequency' }}
+              />
+            </EuiFlexItem>
+            <EuiFlexItem grow={false} style={{ marginLeft: '4px', marginRight: '4px' }}>
+              <FormikFieldNumber
+                name={`actions.${index}.throttle.value`}
```

Comment:
Existing actions will not have a throttle: { value, unit } so this will attempt to read throttle off the action which will be undefined and then value off undefined.

How does formik handle bad paths?

Reply:
Good point. Will test that case and make code change if necessary.